Teams update (#163)
pretix/pretix-docs: docs/guides/teams.md

Having navigated to :navpath:Your organizer → :fa3-users: Teams:, click the :btn-icon:fa3-plus: Create a new team: button in order to create a new team.

What you name your new team and which permissions you grant it is up to you.

> For example, you could name your team "Staff" to distinguish it from the administrators. Under the "Organizer permissions" section you can either select the check box "All organizer permissions" and give it all permissions or grant the following individual organizer permissions: Set "Events" to "Access existing and create new events", "Gift cards" to "View and change", and "Customers" to "View and change".

Reviewer: > For example, you could name your team "Staff" to distinguish it from the administrators. Under the "Organizer permissions" section you can either select the check box "All organizer permissions" and give it all permissions or grant the following individual organizer permissions: Set "Events" to "Access existing and create new events", "Gift cards" to "View and change", and "Customers" to "View and change".

Hier ist das Prinzip "ein Satz pro Zeile" ein bisschen auseinandergebrochen.
Könntest du das einmal reparieren, damit das ein bisschen weniger Textwand ist und die inhaltliche Änderung leichter zu erfassen?

Reviewer: ... und in dem gesamten deutschen Text ebenso, aber ich tippe da jetzt nicht den gleichen Kommentar zwanzigmal. 
Tiefergehendes review erst, wenn die Formatierung repariert ist; sonst ist das wirklich anstrengend und unübersichtlich.

Reviewer 2: Ich war mal so frei, alle Sätze in ihre eigene Zeile zu schieben. Review sollte daher jetzt einfacher sein :)

Author: Danke Matthias! Es ist ein Lernprozess: Die Sache mit ein Satz pro Zeile hatte ich nicht mehr auf dem Schirm.

Having navigated to :navpath:Your organizer → :fa3-users: Teams:, click the :btn-icon:fa3-plus: Create a new team: button in order to create a new team.

What you name your new team and which permissions you grant it is up to you.
For example, you could name your team "Staff" to distinguish it from the administrators.

> Under the "Organizer permissions" section you can either select the check box "All organizer permissions" and give it all permissions or grant the following individual organizer permissions:

Reviewer 2: ```suggestion
If you want to give your team access to all organizer-level permissions, then, under the "Organizer permissions" heading, check the box next to "All organizer permissions". 
If you want to give your team only some of those permissions, uncheck that box. 

A box will appear that allows you to set granular permissions for various aspects of the organizer account. 
```

> Set "Events" to "Access existing and create new events", "Gift cards" to "View and change", and "Customers" to "View and change".

Reviewer 2: ```suggestion
For instance, you could set "Events" to "Access existing and create new events", "Gift cards" to "View and change", and "Customers" to "View and change".
```

The option "Settings" applies to all settings which are nested under :navpath:Your organizer: and which are not controlled by any of the other options on this page.
If the URL of the settings page for which you want to grant or revoke permissions includes `/control/organizer/`, then it is controlled by this option.
Again, this only applies to settings which are not controlled by any of the other checkboxes under "Organizer permissions".

Under the "Event permissions" section you could give the team access to "All events (including newly created ones)" and either grant all event permissions or grant it the following permissions:

> "General settings" to "View and change", "Products quotas and questions" to "View and change", "Orders" to "View all", "Vouchers" to "View and change".

Reviewer 2: This is not how that page works. There are two different things to here: 
1. Grant permissions for different events. Check the box "All events (including newly created ones)" or uncheck it and grant permission for individual events
2. Grant permissions for what to do for those events. Check the box "All event permissions" or uncheck it and grant more granular permissions through the box that appears. 
Please update this section accordingly

> The option "General settings" under event permissions applies to all settings which are nested under :navpath:Your event: and which are not controlled by any of the other options on this page.

Reviewer 2: ```suggestion
The option "General settings" under event permissions applies to all settings which are nested under :navpath:Your event: and which are not controlled by any of the other options on this page.
Among other things, it applies to plugin settings.
```

If the URL of the settings page for which you want to grant or revoke permissions includes `/control/event/`, then it is controlled by this option.

> Again, this only applies to settings which are not controlled by any of the other checkboxes under "Event permissions" including plugin settings.

Reviewer 2: If you commit the previous suggestion, remove this: 
```suggestion
Again, this only applies to settings which are not controlled by any of the other checkboxes under "Event permissions".
```

The problem with this addition is that the scope is not clear. Does it apply to plugin settings or are the plugin settings an exception? This is also a problem with the UI text in English
